### PR TITLE
DTSBP-97 Upgrade terraform azurerm provider

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBP-97

### Change description ###
Upgrade terraform azurerm provider
Change made as suggested in https://tools.hmcts.net/jira/browse/DTSBP-97

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/rpe-send-letter-service/Chart.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 home: https://github.com/hmcts/send-letter-service
-version: 0.2.0
+version: 0.2.1
 description: HMCTS Send letter service
 maintainers:
   - name: HMCTS BSP Team
     email: bspteam@hmcts.net
 dependencies:
   - name: java
-    version: ~3.0.0
```
Comment (jasonpaige) [inline charts/rpe-send-letter-service/Chart.yaml:11]: I _think_ we can actually leave this change for another PR to save complexity. I don't think this has an impact on the azurerm provider

Comment (rishikrsharma) [reply charts/rpe-send-letter-service/Chart.yaml:11]: Ok, I will leave this change

**Review by jasonpaige — APPROVED** [review]
